subworkflows/nf-core/fastq_taxonomic_profile_metaphlan/meta.yml

```
@@ -0,0 +1,52 @@
+name: "fastq_taxonomic_profile_metaphlan"
+description: Subworkflow to taxonomically classify metagenomic sequencing data using MetaPhlAn
+keywords:
+  - metaphlan
+  - metagenomics
+  - database
+  - classification
+  - merge
+  - profiles
+components:
+  - metaphlan/makedb
+  - metaphlan/metaphlan
+  - metaphlan/mergemetaphlantables
+input:
```

```suggestion

input:
```
Personal preference, I'll leave it to you